server/server.go

```diff
@@ -92,10 +92,10 @@ func (s *ipfsLiteServer) AddFile(srv pb.IpfsLite_AddFileServer) error {
 	for {
 		req, err := srv.Recv()
 		if err == io.EOF {
-			writer.Close()
+			_ = writer.Close()
```

Comment:
generally a good idea to explicitly ignore unhandled errors (your IDE linter should be underlining these)